### ISSUE TITLE
[2.1.1] Unable to start after install

### ISSUE THREAD

**Issue by fir3wall** [linagora/linshare#32]
Hello,

I've followed installation steps for Centos 7 setup of most recent Linshare version. Got a problem starting war file, can somebody possibly assist me on this?

https://pastebin.com/A0i8e6Hr

Please note all of the steps worked up without an error.

Thanks
Dariusz

**Comment by chezmohamed** [comment]
I have the same problem that you someone there to the solution?

**Comment by fmartin-linagora** [comment]
Hello,
the first error I see is : 
```
 Cannot resolve reference to bean 'thumbnailGeneratorService' while setting constructor argument; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'thumbnailGeneratorService' defined in file [/var/lib/tomcat/webapps/linshare/WEB-INF/classes/springContext-business-service.xml]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: officeHome doesn't exist or is not a directory:
```
It probably means you did not install libreOffice for the thumbnail generator. you can disable it if you dont want to install it. Every thing is explained here : https://github.com/linagora/linshare/blob/master/documentation/EN/installation/linshare-install.md#thumbnail
Fred.

**Comment by fir3wall** [comment]
@fmartin-linagora 
Thanks will give it a try - I do not require thumbail generation so completely skipped the section, maybe there was something there to set up!

D

**Comment by fmartin-linagora** [comment]
I think thumbnail generator is enabled by default : linshare.documents.thumbnail.enable=true

**Comment by fir3wall** [comment]
@fmartin-linagora 
Fair point fixed my issue after setting it to false. Many thanks for help.

Can you clarify something on S3 support please?

I've understand following options are required:
* linshare.documents.storage.mode=swift-keystone
* linshare.documents.storage.bucket=<that's buket name yeah ? or some magic ID>
* linshare.documents.storage.swift.identity=< is that Access Key ID? >
* linshare.documents.storage.swift.credential=< is that AWS Secret Access Key?>
* linshare.documents.storage.swift.endpoint=<assuming that stays default on http://127.0.0.1:5000/v2.0 >

Thanks
Dariusz

**Comment by fmartin-linagora** [comment]
Thumbnail generator will be disable by default for the next release. 
For S3 support, we have an issue, configuration keys are missing. I will fix it and test it ASAP, probably during august. Sorry. I will keep you posted.

**Comment by fir3wall** [comment]
Many thanks
Dariusz
